README.md

```diff
@@ -7,51 +7,93 @@
+### <img src="https://www.vectorlogo.zone/logos/archlinux/archlinux-icon.svg" height="20"/> Arch Linux
 
-### Building
-If you want to build PolyMC yourself, check [BUILD.md](BUILD.md) for build instructions.
+There are several AUR packages available:  
+[![polymc](https://img.shields.io/aur/version/polymc?label=aur/polymc)](https://aur.archlinux.org/packages/polymc/)  
+[![polymc-bin](https://img.shields.io/aur/version/polymc-bin?label=aur/polymc-bin)](https://aur.archlinux.org/packages/polymc-bin/)  
+[![polymc-git](https://img.shields.io/aur/version/polymc-git?label=aur/polymc-git)](https://aur.archlinux.org/packages/polymc-git/)
 
-You can build the flatpak using [this source](https://github.com/flathub/org.polymc.PolyMC).
+```sh
+# stable source package
+yay -S polymc
+# stable binary package:
+yay -S polymc-bin
+# latest git package:
+yay -S polymc-git
+```
 
-### Code formatting
-Just follow the existing formatting.
+A prebuilt Arch package is available:  
+[Arch Linux (AMD64)](https://packages.polymc.org/latest/arch/polymc-bin-latest-1-x86_64.pkg.tar.zst) ([SHA256](https://packages.polymc.org/latest/arch/polymc-bin-latest-1-x86_64.pkg.tar.zst.sha256))
 
-In general, in order of importance:
-* Make sure your IDE is not messing up line endings or whitespace and avoid using linters.
-* Prefer readability over dogma.
-* Keep to the existing formatting.
-* Indent with 4 space unless it's in a submodule.
-* Keep lists (of arguments, parameters, initializers...) as lists, not paragraphs. It should either read from top to bottom, or left to right. Not both.
+```
+wget https://packages.polymc.org/latest/arch/polymc-bin-latest-1-x86_64.pkg.tar.zst
+sudo pacman -U polymc-bin-latest-1-x86_64.pkg.tar.zst
+```
 
-## Translations
-TODO
+### <img src="https://www.vectorlogo.zone/logos/debian/debian-icon.svg" height="20" /> Debian
 
-## Forking/Redistributing/Custom builds policy
-Do whatever you want, we don't care. Just follow the license. If you have any questions about this feel free to ask in an issue.
+A prebuilt deb package is available:  
+[Debian (AMD64)](https://packages.polymc.org/latest/deb/polymc-amd64.deb) ([SHA256](https://packages.polymc.org/latest/deb/polymc-amd64.deb.sha256))
+
+```sh
+wget https://packages.polymc.org/latest/deb/polymc-amd64.deb
+sudo apt install ./polymc-amd64.deb
+```
+
+### <img src="https://www.vectorlogo.zone/logos/nixos/nixos-icon.svg" height="20" /> Nix
+
+A [Nix derivation](packages/nix/NIX.md) is available.
+
+### <img src="https://www.gentoo.org/assets/img/logo/gentoo-signet.svg" height="20" /> Gentoo
+
+A Gentoo ebuild is available in the [swirl](https://git.swurl.xyz/swirl/ebuilds) overlay, named `games-action/polymc`.
+
+```sh
+emerge --oneshot eselect-repository
+eselect-repository enable swirl
+emaint sync -r swirl
+emerge polymc
+# to use latest git version:
+sudo tee -a /etc/portage/package.accept_keywords <<< "=games-action/polymc-9999 ~amd64"
+```
```

Comment:
Mention that these should be done as root.

Also, I actually added polymc-9999, and the keyword to unmask is `**`, not `~amd64`, per my ebuild standards.